### ISSUE TITLE
Tool-tips are inaccessible

### ISSUE THREAD

**Issue by aardrian** [kazzkiq/balloon.css#78]
1. The tool-tips are inaccessible for keyboard users. They should be triggered on focus as well as on hover.

2. You can make it screen reader accessible by changing the `data-balloon` attribute to `aria-label`, with the caveat that it will completely override the text in the control.

Some resources that can be helpful:

- Example of an accessible CSS-only tool-tip on a form control: https://jsfiddle.net/russambiata/55L1jcwb/
- Accessible Emoji (for non-form controls): https://tink.uk/accessible-emoji/
- Combining the two previous techniques, and making it work in Windows High Contrast Mode: http://adrianroselli.com/2016/12/accessible-emoji-tweaked.html
- The ARIA tool-tip pattern, though overkill for what you are doing: https://www.w3.org/TR/wai-aria-practices-1.1/#tooltip

**Comment by kazzkiq** [comment]
I believe @maximelebreton's PR #56 addresses most of those points.

Will review the code and fix conflicts in order to push it into `v0.6.0`.

**Comment by aardrian** [comment]
Dammit. I missed those issues (#19 and #55 as well). Apparently I suck at GitHub searching.

**Comment by kazzkiq** [comment]
To be honest, the number of accessibility-related issues being open only shows how important this topic is.

I'm closing this to keep this topic centered at #19.
